src/runtime/virtcontainers/qemu_amd64_test.go

```diff
@@ -272,6 +272,26 @@ func TestQemuAmd64AppendProtectionDevice(t *testing.T) {
 	assert.Error(err)
 	assert.Empty(bios)
 
+	// snp protection
+	amd64.(*qemuAmd64).protection = snpProtection
+
+	devices, bios, err = amd64.appendProtectionDevice(devices, firmware, "")
+	assert.NoError(err)
+	assert.Empty(bios)
+
+	expectedOut = append(expectedOut,
+		govmmQemu.Object{
+			Type:            govmmQemu.SNPGuest,
+			ID:              "snp",
+			Debug:           false,
+			File:            firmware,
+			CBitPos:         cpuid.AMDMemEncrypt.CBitPosition,
+			ReducedPhysBits: 1,
+		},
+	)
```

Comment:
```suggestion
	}
```

Reply:
And then remove `:` from https://github.com/kata-containers/kata-containers/pull/5328/files/7de2cecfff27abcd7c7a4190c07dac20700c2cbd#diff-a267a8bb05cd3190291f375b40fa78e822b06986f6d9e88a1d68c1a0774c9447R302